framework/test/fixtures/blocks.ts

```diff
@@ -46,7 +46,7 @@ const getKeyPair = (): { publicKey: Buffer; privateKey: Buffer } => {
 
 export const encodeValidBlockHeader = (header: BlockHeader): Buffer => {
 	const chain = new Chain({
-		accountAsset: {},
+		accounts: defaultAccountSchema,
```

Comment:
If its account schema, can we rename the param name as well?

Reply:
Since constractor affects subsequent PRs, I added the change on https://github.com/LiskHQ/lisk-sdk/issues/5639